bin/templates/cordova/lib/check_reqs.js

```diff
@@ -204,9 +204,11 @@ module.exports.check_java = function () {
             }, () => {
                 var msg =
                 'Failed to run "javac -version", make sure that you have a JDK installed.\n' +
```

Comment:
Should it specifically say v8 / 1.8.* in this line as well?

Reply:
Done as an additional change in PR #619.